src/Snapshot/Trigger/EventCountTrigger.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the broadway/broadway-snapshotting package.
+ *
+ * (c) Qandidate.com <opensource@qandidate.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Broadway\Snapshotting\Snapshot\Trigger;
+
+use Broadway\EventSourcing\EventSourcedAggregateRoot;
+use Broadway\Snapshotting\Snapshot\Trigger;
+
+class EventCountTrigger implements Trigger
+{
+    /**
+     * @var int
+     */
+    private $eventCount = 100;
```

Comment:
We measured performance boost of snapshots and it is huge! I'd default it to let's say 20